src/Microsoft.ML.AdditionalLearners/Microsoft.ML.AdditionalLearners.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeInPackage>Microsoft.ML.AdditionalLearners</IncludeInPackage>
+	<AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
>	 [](start = 0, length = 1)

HERESY!

Reply:
how do you spot them in codeflow !!

---
In reply to: [206356521](https://github.com/dotnet/machinelearning/pull/611#discussion_r206356521) [](ancestors = 206356521)

Reply:
on top right corner next  to (show both) (on right side) is "a.b" button. click on it :)

---
In reply to: [206356725](https://github.com/dotnet/machinelearning/pull/611#discussion_r206356725) [](ancestors = 206356725,206356521)